app/src/androidTest/java/com/sdp/movemeet/view/chat/ChatActivityTest.java

```diff
@@ -50,14 +55,18 @@
 
     private FirebaseAuth fAuth;
     public static final String CHAT_MESSAGE = "my message";
+    private static final String TEST_EMAIL = "movemeet@gmail.com";
+    public static final String TEST_PASSWORD = "password";
+
+    private BackendManager<Message> messageManager;
 
     @Before
     public void signIn() {
         CountDownLatch latch = new CountDownLatch(1);
 
         fAuth = AuthenticationInstanceProvider.getAuthenticationInstance();
 
-        fAuth.signInWithEmailAndPassword("movemeet@gmail.com", "password").addOnCompleteListener(new OnCompleteListener<AuthResult>() {
+        fAuth.signInWithEmailAndPassword(TEST_EMAIL, TEST_PASSWORD).addOnCompleteListener(new OnCompleteListener<AuthResult>() {
```

Comment:
Nice for use general attributs for email and password

Reply:
Yes, this is cleaner! If we can do something like that in most of our test files for the final submission, that would be great!